tests/rados/test_pg_autoscale_flag.py

```diff
@@ -47,16 +49,18 @@ def run(ceph_cluster, **kw):
         with open(pool_configs_path, "r") as fd:
             pool_configs = yaml.safe_load(fd)
 
-        pool_conf = pool_configs["replicated"]["sample-pool-1"]
-        create_given_pool(rados_obj, pool_conf)
-        rados_obj.set_pool_property(
-            pool=pool_conf["pool_name"], props="pg_autoscale_mode", value="warn"
-        )
-        pool_conf = pool_configs["erasure"]["sample-pool-1"]
-        create_given_pool(rados_obj, pool_conf)
-        rados_obj.set_pool_property(
-            pool=pool_conf["pool_name"], props="pg_autoscale_mode", value="off"
-        )
+        if create_re_pool:
+            pool_conf = pool_configs["replicated"]["sample-pool-1"]
+            create_given_pool(rados_obj, pool_conf)
+            rados_obj.set_pool_property(
+                pool=pool_conf["pool_name"], props="pg_autoscale_mode", value="warn"
```

Comment:
Any reason why we are setting the autoscaler to warn here and off in case of EC pool?

Reply:
This is the test for autoscale flag. So there can be 3 modes for autoscaler, ON, OFF, WARN. I wanted to have pools in all the states. By default, all pools are set to ON.

That's the reason, when i'm creating new pools, I am creating one with OFF, and another with WARN, so that we have pools on the cluster with all allowed values of autoscaler.